package.json

```diff
@@ -89,9 +92,10 @@
     "build:firebase": "dotenv -e env/.env.firebase npm run build",
     "build:turk-prolific": "dotenv -e env/.env.turk-prolific npm run build",
     "rebuild": "electron-rebuild",
-    "package:windows": "react-scripts build && electron-forge make --arch x64 --targets @electron-forge/maker-squirrel",
-    "package:linux": "react-scripts build && electron-forge make --arch x64 --targets @electron-forge/maker-deb",
-    "package:mac": "react-scripts build && electron-forge make --arch x64 --targets @electron-forge/maker-dmg"
+    "package": "npm run build && electron-forge make",
+    "package:windows": "npm run package -- --arch x64 --targets @electron-forge/maker-squirrel",
+    "package:linux": "npm run package -- --arch x64 --targets @electron-forge/maker-deb",
+    "package:mac": "npm run package -- --arch x64 --targets @electron-forge/maker-dmg"
```

Comment:
Looks like this does not support ARM macs natively right? Is there an option that supports a `universal` build?

Reply:
Looks like there is a universal value now, I'll swap it in!